doc/nrf/libraries/others/nrf_profiler.rst

```diff
@@ -11,63 +11,63 @@ The Profiler provides an interface for logging and visualizing data for performa
 You can use the module to profile :ref:`app_event_manager` events or custom events.
 The output is provided using RTT and can be visualized in a custom Python backend.
 
-See the :ref:`profiler_sample` sample for an example of how to use the Profiler.
+See the :ref:`nrf_profiler_sample` sample for an example of how to use the Profiler.
 
-.. _profiler_configuration:
+.. _nrf_profiler_configuration:
 
 Configuration
 *************
 
-Since Application Event Manager events are converted to profiler events, the Profiler can be configured to profile custom events or Application Event Manager events, or both.
+Since Application Event Manager events are converted to nrf_profiler events, the Profiler can be configured to profile custom events or Application Event Manager events, or both.
```

Comment:
```suggestion
Since Application Event Manager events are converted to nRF Profiler events, the profiler can be configured to profile custom events or Application Event Manager events, or both.
```